server/sonar-server/src/main/java/org/sonar/server/platform/monitoring/SonarQubeMonitor.java

```diff
@@ -19,30 +19,45 @@
  */
 package org.sonar.server.platform.monitoring;
 
+import com.google.common.base.Function;
```

Comment:
the identity providers are not visible in the api/system/info response. Is it expected?